app/assets/javascripts/dashboard/dashboard_view.js

```diff
@@ -21,11 +21,13 @@ import { parseAsMaybe } from "libs/utils";
 const TabPane = Tabs.TabPane;
 
 const validTabKeys = ["datasets", "advanced-datasets", "tasks", "explorativeAnnotations"];
+const tabKeyToURL = ["gallery", "datasets", "tasks", "annotations"];
```

Comment:
See my other comment for how to define the mapping with an object. If you need both directions (which seems to be the case, see line 192), you can use `_.invert` (https://lodash.com/docs/4.17.10#invert) to construct the "inverse mapping" object.